tests/unit/src/main/scala/tests/TestingServer.scala

```diff
@@ -841,6 +841,20 @@ final class TestingServer(
     } yield TextEdits.applyEdits(textContents(filename), textEdits.toList)
   }
 
+  def codeLensDirect(filename: String): Future[String] = {
```

Comment:
Why not use the existing `codeLenses` method ?

Reply:
hmm, now I can return back to existing one
At the begging I wanted to test that there no lenses returned which leads to failure if existing one is used. But then changed test data.

Reply:
that would be cool, since that class has already way more methods than needed.